configs/system/base.js

```diff
@@ -91,11 +92,11 @@ module.exports = {
           },
           prettyFile: {
             level: 'silent',
-            path: '/tmp/base-drive-pretty.log',
+            path: path.join(os.tmpdir(), `/base-drive-pretty.log`),
```

Comment:
@ICJR I tried running your changes after encountering the same path issue. Found that this file missed out the import for the `path` module

Reply:
@ICJR please make this change in `configs/system/testnet.js` as well